core/cas-server-core-web/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation libraries.nimbus
     implementation libraries.cassecurityfilter
     implementation libraries.thymeleaf
+    implementation libraries.snakeyaml
```

Comment:
Why this?

Reply:
```bc``` wasn't working (after my last merged PR) due to me excluding the spring-boot-starter dependency and this fixed it, but it may not be the right place. (CAS/SpringBoot won't start up without snakeyaml when it tries to process application.yml). I was experimenting with a cas.war with lots of dependencies and something was bringing in logback in despite most dependencies in CAS excluding the spring-boot-starter-logging dependency. dependencyInsight on logback was pointing at spring-boot-starter-logging by way of spring-boot-starter so I was trying to exclude it and specify anything we need from there as needed. I think snakeyaml is the only thing we get from there (that we wouldn't already have). 

It seems like as a rule we exclude almost all transitive dependencies and specify what we need?

Reply:
I am going to go ahead and merge this. It fixes bc and I can move it to a better place later if necessary.